features/tutorial.feature

```diff
@@ -182,16 +182,20 @@ Feature: Following the tutorial
         }
 
         index(req, res) {
-          this.send('todo.list', {}, (todos) => {
-            res.render('index', {todos})
+          this.send('todo.list', {}, (messageName, payload) => {
+            if (messageName === 'todo.listing') {
+              res.render('index', {todos: payload})
+            } else {
+              res.sendStatus(500)
+            }
```

Comment:
Were these changes meant to be included as well?

Reply:
Yeah. They were a result of a change in exorelay